### PR TITLE
Allow SameSite cookie prior to PHP 7.3 #18006

### PR DESCRIPTION
**Pull request by scottix**
Not sure how we can test setcookie other than making sure it doesn't crash

| Q             | A
| ------------- | ---
| Is bugfix?    | ❌
| New feature?  | ❌
| Breaks BC?    | ❌
| Tests pass?   | ✔️
| Fixed issues  | #18006

### CODE REVIEW

File: tests/framework/web/ResponseTest.php [hunk 1]
```diff
@@ -311,4 +311,21 @@ public function testSendFileWithInvalidCharactersInFileName()
...
+            'name'     => 'test',
+            'value'    => 'testValue',
+            'sameSite' => \yii\web\Cookie::SAME_SITE_STRICT,
+        ]));
+
+        ob_start();
+        $response->send();
+        $content = ob_get_clean();
+
+        // Only way to test is that it doesn't create any errors
```
Comment (samdark) [inline tests/framework/web/ResponseTest.php:328]: Would it write anything to headers_list()?

Comment (scottix) [reply tests/framework/web/ResponseTest.php:328]: No headers are handled separately. setcookie is a php function that gets handled by webserver. Unles you have a way to make a test do a legitimate request I don't see a way. I did test the code on our application and worked. I'll leave it up to you to resolve.

Comment (samdark) [reply tests/framework/web/ResponseTest.php:328]: OK.

**Review by samdark — APPROVED** [review]

**Comment by samdark** [comment]
Merged!

**Pull request merged**